Proposals/x-gov-42.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Project Galapago - NFT Lending Protocol
+author: Jack Hui (@jack-jackhui), Vivian Wang <vivian@galapago.app>
+company_name: Poject Galapago
+category: dApps
+focus_area: Defi
+open_source: Yes
+amount_requested: 1255000
```

Comment:
Did you notice that it is highly unlikely that 62% of xGovs will vote for your project as Folks finance requested 1000000 for basically same thing you are describing?

Reply:
@scholtz , Thanks for the comment. I apprecaite the opportunity to clarify. 
First of all, I don't think it is the same. The other prposal is building a peer to peer for 1M ALGO with 13 weeks timeframe. Our first milstone is Peer to Pool instant lending with 10 weeks. If you look at milstones from the other prposal, they are asking 1.7M ALGO for the same capability of instant peer to pool lending in 2025. This makes our proposal stands out: 

- We need less funding(1.25 VS 1.7) to provide the same (and even better) NFT lending. 
- We can propvide this capability before end of calendar year 2023 vs you get it in 2025 and is more expensive. 
- We have a working testnet running already that came through Algorand Foundation APAC accelerator program VS other proposal needs to build from sractch.

Lastly, a vibrant and sustainanle ecosystem is more important than one with a monopoly in a true decentralised environment. I am sure that established players are much more equipped in resources even without the xGov grant to actually build the project out if they are really keen and want to contribute to the overall ecosystem. New players like us been working on the concept for more than 12 months, came through the Algorand accelerator program, with a testnet working app but just need some kick-start funding to actually launch it on the mainnet. Basically this is our plead to the community and leave it to their hands to make a judgement.

Reply:
I mean the Algorand foundation put 2 mil. algos on the table for this first grant round to distribute to all grants. In order your proposal to pass, it will require 62% of all xGov votes which is hardly possible because the Folks finance seems to target to take 50% of all votes.

Reply:
@scholtz , Thanks for the comment. After discussing internally, we have decided to update the proposal to reduce the amount asking for milestone 1 and split the cost to a new milestone. We have carefully reviewed the timeline to make sure that community can still get the same benefit because the added milestone does not extend the mainnet launch timeline as compared to the original proposal.